### ISSUE TITLE
adb connection is lost

### ISSUE THREAD

**Issue by oksep** [Jacksgong/okcat#9]
**device** & **adb** is connected
 
when I execute cmd `okcat  -y darwin.yml` , I got 

```
using config on /Users/sep/.okcat/darwin.yml
using config on /Users/sep/.okcat/logx.yml
find regex: ['data', 'time', 'process', 'thread', 'level', 'tag', 'message'] with (.\S*) *(.\S*) *(\d*) *(\d*) *([A-Z]) *([^:]*): *(.*?)$
ADB CONNECTION IS LOST.
```
How to resolve this issue, tks

**Comment by Jacksgong** [comment]
1. check `adb devices`
2. it's no need to use the extension of the config file( `darwin.ym` -> `darwin`)

**Comment by oksep** [comment]
**adb is connected** , sadly `okcat -y=darwin` got the same result

**Comment by oksep** [comment]
https://github.com/Jacksgong/okcat/pull/10   plz Review the PR  :)

**Comment by Jacksgong** [comment]
Nice job! PR has been Merged and released with [v1.1.5](https://github.com/Jacksgong/okcat/releases/tag/v1.1.5)
